tensorflow_lite_support/python/task/audio/core/tensor_audio.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TensorAudio class."""
+
+import numpy as np
+
+from tensorflow_lite_support.python.task.audio.core.pybinds import _pywrap_audio_buffer
+from tensorflow_lite_support.python.task.audio.core import audio_record
+
+_CppAudioBuffer = _pywrap_audio_buffer.AudioBuffer
+_CppAudioFormat = _pywrap_audio_buffer.AudioFormat
+_LoadAudioBufferFromFile = _pywrap_audio_buffer.LoadAudioBufferFromFile
+
+
+class TensorAudio(object):
+  """A wrapper class to store the input audio."""
+
+  def __init__(self,
```

Comment:
Let's change the approach in the storage mode of TensorAudio a little bit:
* TensorAudio will only store data in a numpy array buffer. It's the responsibility of each task to convert TensorAudio into a `_pywrap_audio_buffer.AudioBuffer` to pass to the C++ implementation of each task.
* `create_from_wav_file` loads the audio data into the _pywrap_audio_buffer.AudioBuffer first, then convert to Numpy array and store it. You should use non-copy conversion like [this](https://github.com/tensorflow/tflite-support/blob/5490f096933fe0348d45c8ca8c955e1f158f8971/tensorflow_lite_support/python/task/vision/core/tensor_image.py#L96)
* Rename `sample_count` to `buffer_size` to align with the naming of the C++ layer.